uninstall.php

```diff
@@ -14,7 +14,7 @@
 }
 
 // Load plugin main file to bootstrap infrastructure and add hooks.
-require_once dirname( __FILE ) . '/google-site-kit.php';
+require_once dirname( __FILE__ ) . '/google-site-kit.php';
```

Comment:
@aaemnnosttv maybe we should use just `__DIR__`?
```suggestion
require_once __DIR__ . '/google-site-kit.php';
```

Reply:
Yeah, the result there is the same and a little cleaner so generally I would opt for that as well, but given that this needs to have maximum compatibility with older PHP versions (just like `google-site-kit.php`) I think it's marginally better to keep the older style here. Given that I think `__DIR__` was introduced in PHP 5.3, it's not really a big concern, but I'm guessing this is probably what @felixarntz was thinking here.

Reply:
Ok, not a big deal though. We can use `dirname`.